src/screens/author/index.tsx

```diff
@@ -44,7 +47,11 @@ const AuthorTitlePosts = (props: any) => {
               </div>
             </AuthorCSS>
             {post.map((x, i) => (
-              <Link href={`/blog/${x.slug}`} key={i}>
+              <Link
+                href={`/blog/?blog=${x.slug}`}
```

Comment:
Better naming of the variables.
```suggestion
                href={`/blog/?blog=${post.slug}`}
```